cli/cmd/status.go

```diff
@@ -31,13 +31,13 @@ problems were found.`,
 			return statusCheckResultWasError(os.Stdout)
 		}
 
-		kubeApi, err := k8s.NewK8sAPi(shell.NewUnixShell(), kubeconfigPath, apiAddr)
+		kubeApi, err := k8s.NewK8sAPI(shell.NewUnixShell(), kubeconfigPath, apiAddr)
 		if err != nil {
 			fmt.Fprintf(os.Stderr, "Error with Kubernetes API: %s\n", err.Error())
 			return statusCheckResultWasError(os.Stdout)
 		}
 
-		conduitApi, err := newApiClient(kubeApi)
+		conduitApi, err := newPublicAPIClient(kubeApi, apiAddr)
```

Comment:
But the status command is an exception to the rule, since we do actually want to know if the call to `k8s.NewK8sAPI` failed. In that case, I'd suggest rewriting this block of code as:

```
		kubeApi, err := k8s.NewK8sAPI(shell.NewUnixShell(), kubeconfigPath)
		if err != nil {
			fmt.Fprintf(os.Stderr, "Error with Kubernetes API: %s\n", err.Error())
			return statusCheckResultWasError(os.Stdout)
		}

		var conduitApi pb.ApiClient
		if apiAddr != "" {
			conduitApi, err = public.NewInternalClient(apiAddr)
		} else {
			conduitApi, err = public.NewExternalClient(controlPlaneNamespace, kubeApi)
		}
		if err != nil {
			fmt.Fprintf(os.Stderr, "Error with Conduit API: %s\n", err.Error())
			return statusCheckResultWasError(os.Stdout)
		}
```